Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/AnisotropyWrapper.java

```diff
@@ -222,11 +211,9 @@ private boolean isCalibrationIsotropic() {
 			.averageScale(0, 1), axis.unit(), unit)).distinct().count() == 1;
 	}
 
-	@SuppressWarnings("unchecked")
 	private void matchOps(final Subspace<BitType> subspace) {
-		milOp = (BinaryFunctionOp) Functions.binary(opService, MILGrid.class,
-			List.class, subspace.interval, new AxisAngle4d(), lines,
-			samplingIncrement, new Random());
+		milOp = Functions.binary(opService, MILPlane.class, Vector3d.class,
+			subspace.interval, new AxisAngle4d(), lines, samplingIncrement);
 		final List<Vector3d> tmpPoints = generate(Vector3d::new).limit(
```

Comment:
comment: this spot may be interesting to cite and discuss in the BoneJ paper: the Moreno et al paper we refer to proposes a different way of doing MIL - how would the milOp signature for that way look?

Reply:
Let's discuss this. I'm not entirely sure what you mean.